evmd/cmd/evmd/config/evmd_config.go

```diff
@@ -87,16 +87,17 @@ func GetMaccPerms() map[string][]string {
 	return maps.Clone(maccPerms)
 }
 
+type EVMAppConfig struct {
```

Comment:
in the future should we define this in the EVM library and not in the evmd package?  can make an issue if you think that makes sense